src/webgl/uniforms.js

```diff
@@ -130,6 +130,21 @@ function checkUniformValue(value) {
   return ok;
 }
 
+/**
+ * Create the current uniform state object to be sent to the devtools
+ */
+export const getUniformsObject = (program, uniforms) => {
+  if (!program) {
+    return null;
+  }
+
+  return Object.keys(program._uniformSetters).reduce((out, uniformName) => {
```

Comment:
I replaced it by `for in` already, tell me if I need to use `for of` instead or if it looks fine